components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -411,6 +379,81 @@ bool PublisherInfoDatabase::Find(int start,
   return list;
 }
 
+int PublisherInfoDatabase::Count(int start,
```

Comment:
does it make sense to have start/limit for a count query?

Reply:
@bridiver I used those args for parity with `Find` but you are right that they probably won't be useful, can remove.

Reply:
@bridiver removed